src/background/proxyHandler/proxyHandler.js

```diff
@@ -108,9 +112,27 @@ export class ProxyHandler extends Component {
    */
   processClientStateChanges(vpnState) {
     console.log(`Processing client state change ${vpnState}`);
-    this.#mLocalProxyInfo.value = vpnState.loophole
-      ? [ProxyUtils.parseProxy(vpnState.loophole)]
-      : [];
+    if (this.#mPlatformOs === "linux") {
+      this.#mLocalProxyInfo.value = [
+        {
+          type: "socks",
+          host: "file:/var/run/mozillavpn.proxy",
+          port: 1234,
+        },
+      ];
+    } else if (this.#mPlatformOs === "win") {
+      this.#mLocalProxyInfo.value = [
+        {
+          type: "socks",
+          host: "127.0.0.1",
+          port: 8123,
+        },
+      ];
+    } else if (vpnState.loophole) {
```

Comment:
given we have not sent anything to users out, we should not preemtively collect backcompat code :)